### ISSUE TITLE
Wrong Content-Type for Verify requests

### ISSUE THREAD

**Issue by pablosantiago** [Vonage/vonage-java-sdk#405]
**Describe the bug**
When using the Java SDK to make requests to the Verify API, the calls fails due to wrong `Content-Type` header.
`VerifyEndpoint` creates a HTTP request with `Content-Type` set as `application/json` but adds all parameters as URL parameters. No JSON request body is created.
This results in the following response:

```
{
    "status": "999",
    "error_text": "Could not parse request :: Unparseable JSON"
}
```

**To Reproduce**
```java
final VonageClient client = VonageClient.builder()
        .apiKey("XXXXX")
        .apiSecret("XXXXX")
        .build();

final VerifyClient verifyClient = client.getVerifyClient();
final VerifyRequest request = VerifyRequest.builder("447700900000", "AcmeInc").build();

final VerifyResponse response = verifyClient.verify(request);
System.out.println("response = " + response.getErrorText());
```

**Expected behavior**
A successful response.

**Request/Response**
```
curl --location --request POST 'https://api.nexmo.com/verify/json?api_key=XXXXXX&api_secret=XXXXXXX&number=447700900000&brand=AcmeInc' \
--header 'Content-Type: application/json' \
--header 'Accept: application/json'
```
```
{"status":"999","error_text":"Could not parse request :: Unparseable JSON"}
```

**Vonage SDK Version**
 - Version: 7.0.0

**Java SDK Version**
- Version: 11

**Proxy Server Used**
 * [ ] Yes
 * [ X ] No

**Comment by pablosantiago** [comment]
Removing the `Content-Type` header (still a POST request with `Accept=application/json`), results in:
```<html>

<head>
	<meta http-equiv="Content-Type" content="text/html;charset=ISO-8859-1" />
	<title>Error 400 </title>
</head>

<body>
	<h2>HTTP ERROR: 400</h2>
	<p>Problem accessing /verify/json. Reason:
		<pre>    Bad Request</pre>
	</p>
	<hr /><i><small>Powered by Jetty://</small></i>
</body>

</html>
```

However, changing it to a GET request works.
According to the [documentation](https://developer.vonage.com/api/verify#requests), both GET and POST should work.

**Comment by pablosantiago** [comment]
It works with version 6.4.2 but not with > 6.5.0.

**Comment by xejarque** [comment]
Exactly the same issue upgrading to version 7.0.0

**Comment by SMadani** [comment]
Fixed in 7.1.0 (available now).
